From cdd76a67ce0d37fe74884b73571b142671831a84 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Thu, 21 Jan 2021 22:19:50 +0200
Subject: [PATCH] [SW-0] gaudi: security: remove psoc double include

psoc_global_conf_regs.h was accidentally included several times.
Remove the redundant inclusion and group psoc related includes.

Change-Id: Idc0c1de37b4445f0c2011d7bc70abad52811768a
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/gaudi/security.c b/gaudi/security.c
index 4928341..66ac117 100644
--- a/gaudi/security.c
+++ b/gaudi/security.c
@@ -11,6 +11,7 @@
 #include <cpu_ca53_cfg_regs.h>
 #include <cpu_if_regs.h>
 #include <psoc_global_conf_regs.h>
+#include <psoc_etr_regs.h>
 #include <pcie_wrap_regs.h>
 #include <pcie_core_regs.h>
 #include <mme0_acc_regs.h>
@@ -26,8 +27,6 @@
 #include <sif_rtr_0_regs.h>
 #include <sif_rtr_ctrl_0_regs.h>
 #include <sif_rtr_ctrl_7_regs.h>
-#include <psoc_global_conf_regs.h>
-#include <psoc_etr_regs.h>
 #include <dma_if_w_s_down_ch0_regs.h>
 #include <dma_if_w_s_regs.h>
 #include <sync_mngr_w_s_regs.h>
